clients/ts/FunctionalTests/ts/Common.ts

```diff
@@ -51,7 +60,7 @@ export function eachTransportAndProtocol(action: (transport: HttpTransportType,
     const protocols: IHubProtocol[] = [new JsonHubProtocol()];
     // IE9 does not support XmlHttpRequest advanced features so disable for now
```

Comment:
Clarify this comment: Only run messagepack tests if in Node, or if the browser supports binary content (indicated by the `responseType` property existing)